docs/build/avalanchego-apis/contract-chain-c-chain-api.md

```diff
@@ -164,8 +163,7 @@ eth_maxPriorityFeePerGas() -> {}
 curl -X POST --data '{
     "jsonrpc":"2.0",
     "id"     :1,
-    "method" :"eth_maxPriorityFeePerGas",
-    "params" :{}
+    "method" :"eth_maxPriorityFeePerGas"
```

Comment:
It turns out that "params" :[] should be used for empty value instead of "params":{}. Can you please make this change in the doc?

Reply:
Sure. Done, and successfully tested.